### PR TITLE
Blog: Inside crosswalk for kubernetes

### PR DESCRIPTION
**Pull request by spara**

### CODE REVIEW

File: content/blog/inside-crosswalk-for-kubernetes/index.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
+---
+title: "Inside Crosswalk for Kubernetes"
+date: 2019-11-18
```
Comment (ericrudder) [inline content/blog/inside-crosswalk-for-kubernetes/index.md]:
```suggestion
date: 2019-11-20
```

File: content/blog/inside-crosswalk-for-kubernetes/index.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+Rapid adoption of a complex platform such as Kubernetes presents challenges. For example, enterprise teams can take six to twelve months to on-ramp to Kubernetes, containers, and the cloud before their application goes into production. Once deployed, misconfigured infrastructure accounts for 40 to 50% of [production outages](https://danluu.com/postmortem-lessons/).
+
+Managed Kubernetes clusters, such as AWS Elastic Kubernetes Services (EKS), Azure Kubernetes Services (AKS), Google Kubernetes Engine and Digital Ocean, have become popular because of the complexity of building and maintaining clusters. However, managed Kubernetes solutions are vanilla and require adding and configuring additional resources to deploy an application in production.
+
+## How Crosswalk for Kubernetes Is Organized
+
+Over the course of a year, the Pulumi team has been working with customers and open source users of our infrastructure as code tools for Kubernetes. What we’ve seen are common patterns that we’ve distilled into six distinct playbooks realized as [stacks](https://www.pulumi.com/docs/intro/concepts/stack/).
```
Comment (ericrudder) [inline content/blog/inside-crosswalk-for-kubernetes/index.md]:
```suggestion
We have been working with customers and open source users of our infrastructure as code tools for Kubernetes and we’ve seen several common patterns that we’ve distilled into six distinct playbooks realized as [stacks](https://www.pulumi.com/docs/intro/concepts/stack/).
```

File: content/blog/inside-crosswalk-for-kubernetes/index.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+### Managed Infrastructure
+
+The current trend is to deploy infrastructure on a managed cloud provider. While there are benefits to using managed infrastructure, provisioning is still required to configure a cluster. At minimum this requires building networking infrastructure, storage backends and any number of resources such as CMS, registries, and data pipelines.’
+
+Crosswalk for Kubernetes covers in detail two of the most common tasks: configuring networking and storage. Kubernetes will typically use private subnets for load balances, private subnets for workers and managed Pod networking. Another requirement of Kubernetes is storage which provides data persistence through either shared storage and or volumes for Pods. There are many types of volumes and they vary from cloud provider but can generally be divided into mechanical drives and SSDs and network backed storage such as CephFS or NFS.
```
Comment (ericrudder) [inline content/blog/inside-crosswalk-for-kubernetes/index.md]:
```suggestion
Crosswalk for Kubernetes covers two of the most common tasks, configuring networking and provisioning storage, in detail. Kubernetes will typically use private subnets for load balances, private subnets for workers and managed Pod networking. Another requirement of Kubernetes is storage which provides data persistence through either shared storage and or volumes for Pods. There are many types of volumes and they vary from cloud provider but can generally be divided into mechanical drives and SSDs and network backed storage such as CephFS or NFS.
```

**Comment by joeduffy** [comment]
Odd, I don't see the new post in the Pulumify preview above. Am I the only one?

**Comment by spara** [comment]
> Odd, I don't see the new post in the Pulumify preview above. Am I the only one?

The build is breaking on the link checker

\├─BROKEN─ https://www.tolacapital.com/ (BLC_UNKNOWN)

**Pull request merged**